src/cli.rs

```diff
@@ -51,6 +51,10 @@ enum Command {
         base_url: String,
         #[clap(long, env, default_value = "false")]
         demo_mode: bool,
+        #[clap(long, env)]
+        oauth_google_client_id: String,
+        #[clap(long, env)]
+        oauth_google_client_secret: String,
```

Comment:
Could you please add these two environment variables to the `.env.example` file with a short comment saying what they're used for?